src/TC/src/FfmpegSwDecoder.cpp

```diff
@@ -374,7 +374,7 @@ void FfmpegDecodeFrame::GetParams(MuxingParams& params)
       (1.0 * pImpl->avctx->framerate.num) / (1.0 * pImpl->avctx->framerate.den);
   params.videoContext.codec = FFmpeg2NvCodecId(pImpl->avctx->codec_id);
 
-  switch (pImpl->avctx->sw_pix_fmt) {
+  switch (pImpl->avctx->pix_fmt) {
```

Comment:
Why did I change this? Probably wrong. Strange that the SW decoder has a HW contect

Reply:
needed to pass tests @gedoensmax

Reply:
@gedoensmax should this change be included. passing tests are kind of nice.

Reply:
Yes we include everything needed to pass tests i would say. If there is reasoning to change it back a PR will be filed. But at least we have one system that we know of it runs on.